chef/cookbooks/hyperv/recipes/python_archive.rb

```diff
@@ -11,10 +11,10 @@
   code <<-EOH
 #{node[:sevenzip][:command]} x #{node[:cache_location]}#{node[:python][:archive]} -o#{node[:python][:path]} -r -y
 EOH
-  not_if { ::File.exists?(node[:python][:installed]) }
+  not_if { ::File.exist?(node[:python][:installed]) }
 end
 
-unless registry_value_exists?("HKEY_LOCAL_MACHINE\\SOFTWARE\\Crowbar", {name: "PyWin32Registered", type: :string, data: "PyWin32 Registered"}, :machine)
+unless registry_value_exist?("HKEY_LOCAL_MACHINE\\SOFTWARE\\Crowbar", {name: "PyWin32Registered", type: :string, data: "PyWin32 Registered"}, :machine)
```

Comment:
Metrics/LineLength: Line is too long. [151/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)<br>Style/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)